### PR TITLE
Checks request method type. For POST request, extracts params from b…

### PR DESCRIPTION
**Pull request by rooch84**
…ody.

See #10491

### CODE REVIEW

**Comment by rooch84** [comment]
OK, I don't think Test Groups 10 and 5 are related to my commit. 

Ideally, the oauth1 and oauth2 tests should be expanded to test both GET and POST requests, but it looks like the request is mocked, and I'm not sure how to create it such that so that it still has the `on` function.

**Comment by rooch84** [comment]
Any updates on when this can be pulled?  Anything more required?

**Comment by rooch84** [comment]
It's not clear what's causing Test 8 to fail.  Any suggestions?

File: packages/oauth/oauth_server.js [hunk 1]
```diff
@@ -158,7 +158,35 @@ const middleware = (req, res, next) => {
...
+            }, {});
+            handler(service, params, res);
+          })
+          .catch(err => {
+            Log.warn(`Unable to parse message body: ${err}`);
+          });
+      } else {
+        handler(service, req.query, res);
+      }
   } catch (err) {
```
Comment (jramer) [inline packages/oauth/oauth_server.js]: The code doesn't handle to deliver state to the if on line 198:
`if (req.query.state && err instanceof Error) {`

File: packages/oauth/oauth_server.js [hunk 2]
```diff
@@ -158,7 +158,35 @@ const middleware = (req, res, next) => {
...
+              return arr;
+            }, {});
+            handler(service, params, res);
+          })
+          .catch(err => {
+            Log.warn(`Unable to parse message body: ${err}`);
+          });
+      } else {
+        handler(service, req.query, res);
+      }
```
Comment (jramer) [inline packages/oauth/oauth_server.js]: The code doesn't handle to deliver state to the if on line 198:
`if (req.query.state && err instanceof Error) {`

**Comment by filipenevola** [comment]
I'm closing this PR in favor of https://github.com/meteor/meteor/pull/10751

File: packages/oauth/oauth_server.js (hunk 2 above)
Comment (rooch84) [reply packages/oauth/oauth_server.js]: Bah! Looks like I missed the email. Sorry.

**Comment by rooch84** [comment]
Good.  I'm just glad that I'll no longer need to manage a separate oauth package from now on :-)

**Pull request closed without merging**